src/plugins/chart_expressions/common/index.ts

```diff
@@ -15,4 +15,3 @@ export {
 export type { Simplify, MakeOverridesSerializable, ChartSizeSpec, ChartSizeEvent } from './types';
 export { isChartSizeEvent } from './types';
 export { getColorCategories } from './color_categories';
-export { useSizeTransitionVeil } from './chart_size_transition_veil';
```

Comment:
We are removing the veil? Amazing!!

Reply:
Yeah we put @drewdaemon through so much unnecessary torment just to remove it!! Sorry Drew 😕

Reply:
He was not a fan either 😄